redhat-operators/amq-streams/amq-streams.v1.1.0.clusterserviceversion.yaml

```
@@ -0,0 +1,618 @@
+                    2.1.1=registry.access.redhat.com/amq7/amq-streams-kafka-connect:1.1.0-kafka-2.1.1
+                - name: STRIMZI_KAFKA_CONNECT_S2I_IMAGES
+                  value: |
+                    2.0.0=registry.access.redhat.com/amq7/amq-streams-kafka-connect-s2i:1.1.0-kafka-2.0.0
+                    2.1.1=registry.access.redhat.com/amq7/amq-streams-kafka-connect-s2i:1.1.0-kafka-2.1.1
+                - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
+                  value: |
+                    2.0.0=registry.access.redhat.com/amq7/amq-streams-kafka-mirror-maker:1.1.0-kafka-2.0.0
+                    2.1.1=registry.access.redhat.com/amq7/amq-streams-kafka-mirror-maker:1.1.0-kafka-2.1.1
+                - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-topic-operator:1.1.0
+                - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-user-operator:1.1.0
+                - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-kafka-init:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-zookeeper-stunnel:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-kafka-stunnel:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-entity-operator-stunnel:1.1.0
+                - name: STRIMZI_LOG_LEVEL
+                  value: INFO
+                - name: STRIMZI_CREATE_CLUSTER_ROLES
+                  value: "TRUE"
+                livenessProbe:
+                  httpGet:
+                    path: /healthy
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                readinessProbe:
+                  httpGet:
+                    path: /ready
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                resources:
+                  limits:
+                    cpu: 1000m
+                    memory: 256Mi
+                  requests:
+                    cpu: 200m
+                    memory: 256Mi
+          strategy:
+            type: Recreate
+  customresourcedefinitions:
+    owned:
+    - name: kafkas.kafka.strimzi.io
+      version: v1alpha1
+      kind: Kafka
+      displayName: Kafka
+      description: Represents a Kafka cluster
+      specDescriptors:
+        - description: The desired number of Kafka brokers.
+          displayName: Kafka Brokers
+          path: kafka.replicas
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
```

Sure you don't want that to be editable?

```suggestion
            - 'urn:alm:descriptor:com.tectonic.ui:podCount'
```